website/docs/troubleshooting.md

```diff
@@ -2,6 +2,23 @@
 title: Troubleshooting
 ---
 
+## Copa and Trivy throw errors when Oracle Linux is passed in
+
+Copa supports patching Oracle Linux in two ways:
+
+With a vulnerability scan, `--ignore-errors` must be passed in:
+
+```bash
+patch -r /oracle-7.9-vulns.json -i docker.io/library/oraclelinux:7.9 --ignore-errors
```

Comment:
copa patch?

Reply:
i don't think this is going to patch all vulns, is that right? can we add a note?

Reply:
This will patch all vulnerabilities, but `--ignore-errors` must be passed in due to how Oracle handles CVEs.

Even if an Oracle image is fully patched, Trivy scans will complain about `_fips`. This is known upstream but has unfortunately remained unfixed for a couple years.

Reply:
we can add a note that this will patch all errors aside from the false positives due to Oracle CVE